app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController <ApplicationController
+
+  def index
+    
```

Comment:
Small, but we could take this extra line out.

Reply:
Done!